plottable.d.ts

```diff
@@ -3724,6 +3737,24 @@ declare module Plottable {
              * @returns {Interactions.Key} The calling Key Interaction.
              */
             offKeyPress(keyCode: number, callback: KeyCallback): Key;
+            /**
+             * Adds a callback to be called when the key with the given keyCode is
+             * released and the user is moused over the Component.
```

Comment:
Very much would want to consider if the user needs to be necessarily moused over the Component.

Let's say that I mouse over a Component, press a key, then mouse out, then release.

I think most would expect the callback for release to be called.